### PR TITLE
manifest: Update for regular upmerge 2022.11.03

### PR DESCRIPTION
**Pull request by desowin**
Please submit all things that you want to be included in upmerge to my fork. See https://github.com/desowin/nrfconnect-sdk-nrf/pull/1 for example.

---

Update the following projects:
  * zephyr
  * mcuboot
  * trusted-firmware-m
to align with Zephyr commit adfc705b51643245cb9e46edae614b466b1a571d.

### CODE REVIEW

**Review by martintv — APPROVED** [review]

**Comment by mariuszpos** [comment]
Please add PR for HomeKit repo: https://github.com/nrfconnect/sdk-homekit/pull/446

**Comment by desowin** [comment]
> Please add PR for HomeKit repo: [nrfconnect/sdk-homekit#446](https://github.com/nrfconnect/sdk-homekit/pull/446)

If you want your PR to be included here, open a PR on my fork to branch upmerge.

File: samples/tfm/tfm_hello_world/src/main.c [hunk 1]
```diff
@@ -79,13 +79,13 @@ void main(void)
 	printk("Reading some secure memory that NS is allowed to read\n");
 
 	NRF_TIMER1_NS->TASKS_START = 1;
-	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.PART);
+	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.RAM);
```
Comment (jfischer-no) [inline samples/tfm/tfm_hello_world/src/main.c:82]: @joerchan Should not part variable be renamed to info_ram or removed, and printk() below adapted accordingly?

**Review by SeppoTakalo — APPROVED** [review]

**Review by jorgenmk — APPROVED** [review]

**Review by hakonfam — APPROVED** [review]

File: samples/tfm/tfm_hello_world/src/main.c (hunk 1 above)
Comment (joerchan) [reply samples/tfm/tfm_hello_world/src/main.c:82]: It should, @desowin can you amend the commit?

Comment (desowin) [reply samples/tfm/tfm_hello_world/src/main.c]: fixed

**Review by sachinthegreen — APPROVED** [review]

File: samples/tfm/tfm_hello_world/src/main.c [hunk 2]
```diff
@@ -74,18 +74,18 @@ void main(void)
...
-	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.PART);
+	info_ram = secure_read_word((intptr_t)&NRF_FICR_S->INFO.RAM);
 	NRF_TIMER1_NS->TASKS_CAPTURE[0] = 1;
 	printk("Approximate IPC overhead us: %d\n", NRF_TIMER1_NS->CC[0]);
 
-	printk("FICR->INFO.PART: 0x%08x\n", part);
-	printk("FICR->INFO.VARIANT: 0x%08x\n",
-		secure_read_word((intptr_t)&NRF_FICR_S->INFO.VARIANT));
+	printk("FICR->INFO.RAM: 0x%08x\n", info_ram);
+	printk("FICR->INFO.FLASH: 0x%08x\n",
```
Comment (oyvindronningstad) [inline samples/tfm/tfm_hello_world/src/main.c:87]: The regex in sample.yaml needs to be changed accordingly. (I'm surprised this doesn't fail - is it not run in CI?)

**Review by jhn-nordic — APPROVED** [review]

File: samples/tfm/tfm_hello_world/src/main.c (hunk 2 above)
Comment (magnev) [reply samples/tfm/tfm_hello_world/src/main.c:87]: This does fail in CI

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -617,7 +617,7 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``b96453f653b674629c617bdbc810c3d9f7916ef4``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``cd16a8388f71a6cce0cea871f75f6d4ac8f56da9``, plus some |NCS| specific additions.
```
Comment (anangl) [inline doc/nrf/releases/release-notes-changelog.rst:620]: The Zephyr SHA should be also applied in the following part: https://github.com/nrfconnect/sdk-nrf/blob/b75b91188dbc0ecef7d40ce07c9786bec6130c17/doc/nrf/releases/release-notes-changelog.rst#L626-L634

**Review by anangl — APPROVED** [review]

**Pull request merged**